### PR TITLE
ISSUE-97 Keystore/get method

### PR DESCRIPTION
**Pull request by LanKhuat**

### CODE REVIEW

File: openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraKeystoreGetMethodContract.scala [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+         |      "accountId": "$ACCOUNT_ID",
+         |      "state": "${INSTANCE.value}",
+         |      "list": {}
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin)
+  }
+
+  @Test
+  def keystoreGetShouldRejectIdsField(): Unit = {
```
Comment (chibenwa) [inline openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraKeystoreGetMethodContract.scala:156]: keystoreGetShouldRejectEmptyIdsField

File: openpaas-james/jmap/extensions/src/main/scala/com/linagora/openpaas/james/jmap/json/KeystoreSerializer.scala [hunk 2]
```diff
@@ -28,14 +28,29 @@ class KeystoreSerializer {
...
+  private implicit val stateWrites: Writes[State] = Json.valueWrites[State]
+  private implicit val publicKeyWrites: Writes[PublicKey] = Json.writes[PublicKey]
+  private implicit val keystoreMapGetResponseWrites: Writes[Map[KeyId, PublicKey]] = mapWrites[KeyId, PublicKey](_.value, publicKeyWrites)
+
   private implicit val keyIdReads: Format[KeystoreCreationId] = Json.valueFormat[KeystoreCreationId]
   private implicit val keystoreDestroyReads: Reads[List[KeystoreCreationId]] = Reads.list[KeystoreCreationId]
   private implicit val keystoreSetRequestReads: Reads[KeystoreSetRequest] = Json.reads[KeystoreSetRequest]
   private implicit val keystoreSetResponseWrites: Writes[KeystoreSetResponse] = Json.writes[KeystoreSetResponse]
+  private implicit val keystoreGetRequestReads: Reads[KeystoreGetRequest] = {
+    case jsonObject: JsObject if jsonObject.keys.contains("ids") => JsError("current implementation only supports list all")
```
Comment (chibenwa) [inline openpaas-james/jmap/extensions/src/main/scala/com/linagora/openpaas/james/jmap/json/KeystoreSerializer.scala:41]: Next step of this PR (in follow up commits): add support for the ids property!

**Review by Arsnael — COMMENTED** [review]
Read it, nothing to add so far

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
Can you rebase on master, now that the Keystore/set destroy method has been merged?

**Pull request merged**